client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -126,10 +130,12 @@ const Header: FunctionComponent = () => {
 				placeholder: ! recommendedDomainSuggestion,
 			} ) }
 		>
-			{ recommendedDomainSuggestion
-				? /* translators: domain name is available, eg: "yourname.com is available" */
-				  sprintf( __( '%s is available' ), recommendedDomainSuggestion.domain_name )
-				: 'example.wordpress.com' }
+			{ isMobile && __( 'Domain available' ) }
+			{ ! isMobile &&
+				( recommendedDomainSuggestion
+					? /* translators: domain name is available, eg: "yourname.com is available" */
+					  sprintf( __( '%s is available' ), recommendedDomainSuggestion.domain_name )
+					: 'example.wordpress.com' ) }
```

Comment:
I'd love a beautiful nested ternary here, but the linter will disagree with my sense of aesthetics 🙃

Reply:
also this format helps preserving some basic level of readability once we start adding ternaries to the mobile case 🤷